modules/pam_sepermit/pam_sepermit.c

```diff
@@ -372,13 +374,25 @@ pam_sm_authenticate(pam_handle_t *pamh, int flags UNUSED,
 	char *level = NULL;
 	const char *cfgfile = SEPERMIT_CONF_FILE;
 
+#ifdef VENDORDIR
+	/* If the specified file does not exist, try the vendor file as fallback.
+	   This can be overruled again by a user provided file definition (conf=...) */
+	struct stat buffer;
+	if (stat(cfgfile, &buffer) != 0 && errno == ENOENT) {
+	  cfgfile = VENDORDIR"/security/sepermit.conf";
+	  pam_syslog (pamh, LOG_INFO,
+		      "cannot read settings from %s. Trying %s", SEPERMIT_CONF_FILE, cfgfile);
+	}
+#endif
+
 	/* Parse arguments. */
 	for (i = 0; i < argc; i++) {
+		const char *str;
 		if (strcmp(argv[i], "debug") == 0) {
 			debug = 1;
 		}
-		if (strcmp(argv[i], "conf=") == 0) {
-			cfgfile = argv[i] + 5;
+		if ((str = pam_str_skip_prefix(*argv, "conf=")) != NULL) {
+			cfgfile = str;
```

Comment:
`*argv == argv[i]` if and only if `i == 0`.

Reply:
Yes, I have taken this from pam_wheel.c and have not noticed that it has a different "for" loop. But what do you mean with "if and only if i == 0".?Does it mean that conf= is only valid at the first point of the arguments ?